onnxruntime/test/framework/ort_model_only_test.cc

```diff
@@ -193,6 +193,26 @@ static void CompareGraphAndSessionState(const InferenceSessionWrapper& session_o
   }
 }
 
+static void CompareSessionMetadata(const InferenceSessionWrapper& session_object_1,
+                                   const InferenceSessionWrapper& session_object_2) {
+  const auto pair_1 = session_object_1.GetModelMetadata();
+  ASSERT_STATUS_OK(pair_1.first);
+  const auto& metadata_1 = *pair_1.second;
+
+  const auto pair_2 = session_object_2.GetModelMetadata();
+  ASSERT_STATUS_OK(pair_2.first);
+  const auto& metadata_2 = *pair_2.second;
+
+  ASSERT_EQ(metadata_1.producer_name, metadata_2.producer_name);
+  // ORT format does not have graph name
+  // ASSERT_EQ(metadata_1.graph_name, metadata_2.graph_name);
```

Comment:
Do we want to add this also?

Reply:
Given we can add it if needed in the future without breaking backwards compat, I'd hold off on adding it. 

If someone is desperate they could use the graph description as a fallback.